src/main/io/gps.c

```diff
@@ -440,6 +467,25 @@ void gpsInitNmea(void)
 #endif // USE_GPS_NMEA
 
 #ifdef USE_GPS_UBLOX
+const uint8_t ubloxUTCStandardConfig_int[5] = {
+        UBLOX_UTC_STANDARD_AUTO,
+        UBLOX_UTC_STANDARD_USNO,
+        UBLOX_UTC_STANDARD_EU,
+        UBLOX_UTC_STANDARD_SU,
+        UBLOX_UTC_STANDARD_NTSC
+};
```

Comment:
Looks like you don't need this. Just use the enums `UBX_UTC_STANDARD_x` directly. Enums are basically just integers under the hood in C.
```suggestion
```

Reply:
Suggestion rejected since the integer code for the UTC standard is required, not the value from the position in the enum.